test/extensions/filters/network/common/fuzz/uber_per_readfilter.cc

```diff
@@ -18,27 +18,23 @@ namespace {
 static const int SecondsPerDay = 86400;
 } // namespace
 std::vector<absl::string_view> UberFilterFuzzer::filterNames() {
-  // These filters have already been covered by this fuzzer.
-  // Will extend to cover other network filters one by one.
+  // Add filters that are in the process of being or are robust against untrusted downstream
+  // traffic.
   static std::vector<absl::string_view> filter_names;
   if (filter_names.empty()) {
     const auto factories = Registry::FactoryRegistry<
         Server::Configuration::NamedNetworkFilterConfigFactory>::factories();
     const std::vector<absl::string_view> supported_filter_names = {
-        NetworkFilterNames::get().ExtAuthorization, NetworkFilterNames::get().LocalRateLimit,
-        NetworkFilterNames::get().RedisProxy, NetworkFilterNames::get().ClientSslAuth,
-        NetworkFilterNames::get().Echo, NetworkFilterNames::get().DirectResponse,
-        NetworkFilterNames::get().DubboProxy, NetworkFilterNames::get().SniCluster,
+        NetworkFilterNames::get().ExtAuthorization,
+        NetworkFilterNames::get().LocalRateLimit,
+        NetworkFilterNames::get().ClientSslAuth,
         // A dedicated http_connection_manager fuzzer can be found in
         // test/common/http/conn_manager_impl_fuzz_test.cc
-        NetworkFilterNames::get().HttpConnectionManager, NetworkFilterNames::get().ThriftProxy,
-        NetworkFilterNames::get().ZooKeeperProxy, NetworkFilterNames::get().SniDynamicForwardProxy,
-        NetworkFilterNames::get().KafkaBroker, NetworkFilterNames::get().RocketmqProxy,
-        NetworkFilterNames::get().RateLimit, NetworkFilterNames::get().Rbac,
-        NetworkFilterNames::get().MongoProxy, NetworkFilterNames::get().MySQLProxy
-        // TODO(jianwendong): add "NetworkFilterNames::get().Postgres" after it supports untrusted
-        // data.
-        // TODO(jianwendong): add fuzz test for "NetworkFilterNames::get().TcpProxy".
+        NetworkFilterNames::get().HttpConnectionManager,
+        NetworkFilterNames::get().RateLimit,
+        NetworkFilterNames::get().Rbac,
+        NetworkFilterNames::get().TcpProxy,
+        NetworkFilterNames::get().Echo,
```

Comment:
Could you please alphabetize the list above?

Reply:
done